### DISCUSSION TITLE
Get nutrient totals from diary

### DISCUSSION THREAD

**Discussion by slapcat** [coddingtonbear/python-myfitnesspal#188] [Ideas]
It seems that this library only grabs some of the `nutritional_contents` from the MFP API for meals. It would be great if it could display all, such as cholesterol and fiber. The API docs indicate that this information is included: https://myfitnesspalapi.com/docs/diary-get/

**Comment by coddingtonbear** [comment]
Unfortunately, although the API docs you've referenced are probably correct, this library (like all others) does not use their public API and instead relies on web scraping and other less-elegant approaches.  This is unfortunately necessary due to the fact that it is, as far as I'm aware, not possible to get an API key for the MyFitnessPal API unless you have a business relationship with MyFitnessPal (see "We are not accepting requests for API access at this time." here : https://myfitnesspalapi.com/)
